### ISSUE TITLE
tt-wdio script: whitespaces cannot be used to separate values in the options list

### ISSUE THREAD

**Issue by benbcai** [cerner/terra-toolkit-boneyard#49]
# Bug Report

## Description
When a whitespace is used to separate two individual values in an option list passed to the `tt-wdio` bin script, the latter part of the value is ignored.

In this example, both `en` and `en-US` are recognized values in the locales option.
`"tt-wdio --locales=['en','en-US']"`

However, this example has a white space between the locale values and only recognizes `en`. Any value after the first space are ignored.
`"tt-wdio --locales=['en', 'en-US']"`

The locales option passed to commander looks like this:
`[ 'en', '' ]`

## Additional Context / Screenshots
When the shell detects the whitespace the next item after it is treated as a separate argument/option instead of a separate value in the list before it reaches to the commander module. The above example has the following arguments. Notice `en-US` is treated as a whole separate argument.

1. `/usr/local/bin/node`
2. `../.bin/tt-wdio`
3. `--locales=[en,`
4. `en-US]`

See https://github.com/tj/commander.js/issues/957

## Expected Behavior
The above commander issue mentioned that the shell script is parsing the whitespace into a new argument. I don't know that there is anything the `tt-wdio` can do here. Perhaps, we can add documentation to clearly state that whitespaces are not allowed in the options list.

## Possible Solution
Using double quotes as in `"tt-wdio --locales=[\"'en', 'en-US'\"]"` works but why not just remove the whitespace.

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):

**Comment by emilyrohrbough** [comment]
I think this comes into how the command line accepts and parses arguments. I'm not sure if there is a good way to get around this because a space typically indicates a new argument, command or flag that should be passed to the script. We could probably get creative though we wanted 👍

**Comment by ryanthemanuel** [comment]
Let's add some documentation about how this works.  I don't think it's worth the effort to get around how the command line works.  Possibly in the FAQ.

**Comment by ShettyAkarsh** [comment]
In this scenario because of this issue the list will be passed down as below

`['en', ' ']`

![image](https://user-images.githubusercontent.com/45097016/99392290-7b94fd80-2901-11eb-82e7-b2ddb6e512e2.png)

so we will have to avoid passing/execution of this invalid value down the line. some of the possible fixes are.

1.  Do not pass the Blank value down the line. pass only valid values ['en'].
2. Add the warning [here](https://github.com/cerner/terra-toolkit-boneyard/blob/7339e816967fed5617c23b4d3bb7ef54d354a99b/scripts/wdio/wdio-runner-cli.js) when list values are not passed correctly and stop the process.
3. Exit the WDIO execution for invalid value (i.e [' ']) here [here](https://github.com/cerner/terra-toolkit-boneyard/blob/7339e816967fed5617c23b4d3bb7ef54d354a99b/scripts/wdio/wdio-runner.js#L37-L80).
